debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleSnapshotChangeEventSource.java

```diff
@@ -131,10 +135,6 @@ protected void determineSnapshotOffset(RelationalSnapshotContext<OracleOffsetCon
                 .build();
     }
 
-    private Scn getCurrentScn(SnapshotContext<OracleOffsetContext> ctx) throws SQLException {
```

Comment:
The `ctx` argument is unused here, and the method is used only in one place, so instead of changing the argument type, I inlined the call.

Reply:
Makes sense, thx.